README.md

```diff
@@ -17,6 +17,25 @@ add valid accounts in [apps/ae_socket_connector/lib/test_accounts.ex](apps/ae_so
 
 add the address to your [æternity](https://github.com/aeternity/aeternity) node and your network id in [apps/ae_socket_connector/lib/channel_runner.ex](apps/ae_socket_connector/lib/channel_runner.ex#L4)
 
+## Local node optional configuration
+
+If you host your own node make sure to bump `counter`
+```json
+"sc_ws_handlers" : {
+                    "description" : "State channel websocket 
+                    ...
+                    "properties" : {
+                        ...
+                        "counter" : {
+                            "type" : "integer",
+                            "default" : 100
+                        },
+                        ...
+                    }
+                },
+```
+in apps/aeutils/priv/aeternity_config_schema.json
```

Comment:
Interesting :) Isn't it better to propose changing actual user configuration that changing an internal schema file ? 🤔

Reply:
you are the configuration master, show me how it's done properly

Reply:
Well, when you run a node, you usually have a `aeternity.yaml` in the top directory of the node .. and it seems much better place for configuration than changing the node internal files defaults :)